lib/intersection_validator.rb

```diff
@@ -0,0 +1 @@
+IntersectionValidator = Class.new(ActiveModelValidatesIntersectionOf::Validator)
```

Comment:
Can you use an explicit inheritance in this case? Will be more friendly readable!